runtime/queries/openscad/highlights.scm

```diff
@@ -0,0 +1,51 @@
+(number) @number
```

Comment:
Some of the scopes in these queries need to be replaced with the ones helix uses: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

`@number` should become `@constant.numeric`, the `(boolean) @constant.builtin` stanza can become `@constant.builtin.boolean` and `@delimeter` should become `@punctuation.delimiter`

Reply:
thanks for the feedback. i've renamed and split up the scopes a little.
makes sense and looks better now. still learning hx 😊